### PR TITLE
Some more links, structure adjustment and CoC

### PR DESCRIPTION
**Pull request by cerebrux**
Separated the places that someone would like to get involved, added link to the contribute page and also some info for the Code Of Conduct

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 
-If you want to contribute, you are very welcome on our IRC channels #nextcloud & #nextcloud-dev (on freenode) and our forum at https://help.nextcloud.com – if you want to join the Github organization just let us know and we’ll add you! :)
+Meanwhile check out https://nextcloud.com and follow us on https://twitter.com/nextclouders
+
+If you want to [contribute](https://nextcloud.com/contribute/), you are very welcome: 
+
+- on our IRC channels #nextcloud & #nextcloud-dev irc://#nextcloud-dev@freenode.net (on freenode) and 
+- our forum at https://help.nextcloud.com
+
+Please read the [Code of Condact](https://nextcloud.com/community/code-of-conduct/). This document offers some guidance to ensure Nextcloud participants can cooperate effectively in a positive and inspiring atmosphere, and to explain how together we can strengthen and support each other.
```
Comment (vincchan) [inline README.md]: typo, should be "Code of Conduct".

Comment (cerebrux) [inline README.md]: Thanks !

**Comment by jyaworski** [comment]
@cerebrux can you squash into one commit?

**Comment by jancborchardt** [comment]
Awesome @cerebrux, great on adding the link to the Code of Conduct! :+1: 

(Sure, next time squashing into one commit would be good but for now it’s fine. :) @jyaworski thanks for the quick answer! :)

**Comment by LukasReschke** [comment]
That's awesome! Great work. And nice to have you all in the community @cerebrux @vincchan @jyaworski @jancborchardt ! 🚀 ❤️ 🎉

**Pull request merged**